### PR TITLE
[Major] Updated tokens wizard

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

**Review by aweell — CHANGES_REQUESTED** [review]

File: docs/src/pages/themeBuilder/themes/AdvancedTheme.json [hunk 1]
```diff
@@ -1110,57 +1110,64 @@
...
-    "stepContainerFontSize": "1rem",
-    "stepContainerFontStyle": "normal",
-    "stepContainerFontWeight": "400",
-    "stepContainerLetterSpacing": "0.05em",
-    "stepContainerFontColor": "#000000",
-    "stepContainerBackgroundColor": "#ffffff",
-    "stepContainerSelectedFontColor": "#ffffff",
-    "stepContainerSelectedBackgroundColor": "#5f249f",
-    "stepContainerIconSize": "19px",
+    "disabledStepperBackgroundColor": "#f2f2f2",
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply docs/src/pages/themeBuilder/themes/AdvancedTheme.json]: Done

**Review by aweell — DISMISSED** [review]

File: lib/src/common/variables.js [hunk 2]
```diff
@@ -1286,57 +1286,64 @@ export const componentTokens = {
...
+    disabledStepFontColor: globalTokens.hal_grey_l_60,
+    disabledLabelFontColor: globalTokens.hal_grey_l_60,
+    disabledHelperTextFontColor: globalTokens.hal_grey_l_60,
+    disabledStepBorderColor: globalTokens.hal_grey_l_95,
+    stepFontSize: globalTokens.type_scale_03,
+    stepFontFamily: globalTokens.type_sans,
+    stepFontStyle: globalTokens.type_normal,
+    stepFontWeight: globalTokens.type_regular,
+    stepFontTracking: globalTokens.type_spacing_wide_02,
+    stepFontColor: globalTokens.hal_black,
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply lib/src/common/variables.js]: Done

File: lib/src/common/variables.js [hunk 3]
```diff
@@ -1286,57 +1286,64 @@ export const componentTokens = {
...
-    selectedCircleBorderStyle: "solid",
-    selectedCircleBorderRadius: "45px",
-    selectedCircleBorderColor: globalTokens.purple,
-    disabledCircleWidth: "32px",
-    disabledCircleHeight: "32px",
-    disabledCircleBorderThickness: "2px",
-    disabledCircleBorderStyle: "solid",
-    disabledCircleBorderRadius: "45px",
-    disabledCircleBorderColor: globalTokens.lightGrey,
+    labelFontColor: globalTokens.hal_black,
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply lib/src/common/variables.js]: Done

**Comment by Lisa159** [comment]
Added more rename of tokens:
- From `stepFontColor` to `visitedStepFontColor`.
- From `stepBackgroundColor` to `visitedStepBackgroundColor`.
- From `labelFontColor` to `visitedLabelFontColor`.
- From `activeLabelFontColor` to `selectedLabelFontColor`.
- From `helperTextFontColor` to `visitedHelperTextFontColor`.
- From `activeHelperTextFontColor` to `selectedHelperTextFontColor`.
- From `stepBorderColor` to `visitedStepBorderColor`.

**Review by GomezIvann — APPROVED** [review]

**Pull request merged**